libgrust/libproc_macro/bridge.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2023 Free Software Foundation, Inc.
+//
+// This file is part of the GNU Proc Macro Library.  This library is free
+// software; you can redistribute it and/or modify it under the
+// terms of the GNU General Public License as published by the
+// Free Software Foundation; either version 3, or (at your option)
+// any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// Under Section 7 of GPL version 3, you are granted additional
+// permissions described in the GCC Runtime Library Exception, version
+// 3.1, as published by the Free Software Foundation.
+
+// You should have received a copy of the GNU General Public License and
+// a copy of the GCC Runtime Library Exception along with this program;
+// see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
+// <http://www.gnu.org/licenses/>.
+#ifndef BRIDGE_H
+#define BRIDGE_H
+
+namespace ProcMacro {
```

Comment:
```suggestion

namespace Rust {
namespace ProcMacro {
```

or no? I can't remember haha

Reply:
No, proc macros never used any `Rust` parent namespace. Do you think it would be better with one ?

Reply:
nope, that;s all good for me, just checking!